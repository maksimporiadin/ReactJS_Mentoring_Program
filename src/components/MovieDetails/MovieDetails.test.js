import 'jsdom-global/register';
import React from 'react';

import { MovieDetails } from './index';

import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

configure({adapter: new Adapter()});

describe('MovieDetails', () => {
    let wrapper;
    const defaultProps = {
        movie: {
            poster_path: 'https://movie.url',
            title: 'title',
            tagline: 'This good movie',
            release_date: '2014_12_12',
            runtime: 124,
            overview: 'Nice movie'
        }
    };

    beforeEach(()=>{
        jest.resetAllMocks();
        wrapper = shallow(<MovieDetails {...defaultProps} />)
    });

    it('is renders', () => {
        expect(wrapper).toMatchSnapshot();
    });
});