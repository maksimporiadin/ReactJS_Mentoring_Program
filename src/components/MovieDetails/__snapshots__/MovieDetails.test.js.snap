// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetails is renders 1`] = `
<div
  className="movie-details-container"
>
  <div
    className="movie-details-poster"
  >
    <img
      alt="title"
      className="movie-details-img"
      src="https://movie.url"
    />
  </div>
  <div
    className="movie-details-inform"
  >
    <h1
      className="movie-details-title"
    >
      title
    </h1>
    <h5>
      This good movie
    </h5>
    <div
      className="movie-details-tracks"
    >
      <span>
        2014
      </span>
      <span>
        124
         min
      </span>
    </div>
    <div>
      Nice movie
    </div>
  </div>
</div>
`;
